src/views/esp/view_esp.cpp

```diff
@@ -85,6 +85,85 @@ namespace big
 			if (distance < g.esp.box_render_distance[1] && distance > g.esp.box_render_distance[0] && g.esp.box)
 				draw_list->AddRect({esp_x - (62.5f * multplr), esp_y - (175.f * multplr)}, {esp_x - (62.5f * multplr) + (125.f * multplr), esp_y - (175.f * multplr) + (350.f * multplr)}, esp_color);
 
+			if (distance < g.esp.skeleton_render_distance[1] && distance > g.esp.skeleton_render_distance[0] && g.esp.skeleton)
+			{
+				std::vector<PedBones> bone_ids = // size: 18
+				{
+				    PedBones::SKEL_Head,
+				    PedBones::SKEL_Neck_1,
+				    PedBones::SKEL_Spine_Root,
+				    PedBones::SKEL_Pelvis,
+				    PedBones::SKEL_L_Thigh,
+				    PedBones::SKEL_R_Thigh,
+				    PedBones::SKEL_L_Calf,
+				    PedBones::SKEL_R_Calf,
+				    PedBones::SKEL_L_Foot,
+				    PedBones::SKEL_R_Foot,
+				    PedBones::SKEL_L_Clavicle,
+				    PedBones::SKEL_R_Clavicle,
+				    PedBones::SKEL_L_UpperArm,
+				    PedBones::SKEL_R_UpperArm,
+				    PedBones::SKEL_L_Forearm,
+				    PedBones::SKEL_R_Forearm, 
+					PedBones::SKEL_L_Hand, 
+					PedBones::SKEL_R_Hand
```

Comment:
Different indentations, try to run the code formatter on this file to fix it.

Reply:
Fixed